ERROR - 2025-09-01 08:38:33 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'capstone_db' in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'capstone_db', 3306, '', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'capstone_db' in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2025-09-01 08:49:58 --> mysqli_sql_exception: Table 'users' already exists in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\Capstone\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\Capstone\app\Database\Migrations\2025-08-21-084947_CreateUsersTable.php(47): CodeIgniter\Database\Forge->createTable('users')
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateUsersTable->up()
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-01 09:31:55 --> mysqli_sql_exception: Table 'users' already exists in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\Capstone\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\Capstone\app\Database\Migrations\2025-08-21-084947_CreateUsersTable.php(41): CodeIgniter\Database\Forge->createTable('users')
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateUsersTable->up()
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-01 09:35:06 --> mysqli_sql_exception: Table 'admin' already exists in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `a...', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `a...')
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `a...')
#3 C:\xampp\htdocs\Capstone\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `a...')
#4 C:\xampp\htdocs\Capstone\app\Database\Migrations\2025-08-28-014959_CreateAdminTable.php(41): CodeIgniter\Database\Forge->createTable('admin')
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateAdminTable->up()
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-01 09:37:37 --> mysqli_sql_exception: Table 'capstone_db.users' doesn't exist in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `us...', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `us...')
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `us...')
#3 C:\xampp\htdocs\Capstone\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `us...')
#4 C:\xampp\htdocs\Capstone\app\Database\Migrations\2025-09-01-093103_AddPhoneToUsers.php(11): CodeIgniter\Database\Forge->addColumn('users', Array)
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(876): App\Database\Migrations\AddPhoneToUsers->up()
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-01 09:38:36 --> mysqli_sql_exception: Table 'capstone_db.users' doesn't exist in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `us...', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `us...')
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `us...')
#3 C:\xampp\htdocs\Capstone\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `us...')
#4 C:\xampp\htdocs\Capstone\app\Database\Migrations\2025-09-01-093103_AddPhoneToUsers.php(11): CodeIgniter\Database\Forge->addColumn('users', Array)
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(876): App\Database\Migrations\AddPhoneToUsers->up()
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-01 09:39:45 --> mysqli_sql_exception: Table 'admin' already exists in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `a...', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `a...')
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `a...')
#3 C:\xampp\htdocs\Capstone\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `a...')
#4 C:\xampp\htdocs\Capstone\app\Database\Migrations\2025-08-28-014959_CreateAdminTable.php(41): CodeIgniter\Database\Forge->createTable('admin')
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateAdminTable->up()
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-01 09:44:19 --> mysqli_sql_exception: Table 'users' already exists in C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Capstone\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\Capstone\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\Capstone\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\Capstone\app\Database\Migrations\2025-08-21-084947_CreateUsersTable.php(41): CodeIgniter\Database\Forge->createTable('users')
#5 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateUsersTable->up()
#6 C:\xampp\htdocs\Capstone\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\Capstone\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Capstone\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Capstone\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\Capstone\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\Capstone\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\Capstone\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\Capstone\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\Capstone\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
